src/client.js

```diff
@@ -8,7 +8,8 @@ let client: ?KintoClient;
 
 function getAuthHeader(auth: AuthData): ?string {
   switch (auth.authType) {
-    case "fxa": {
+    case "fxa":
+    case "openid": {
```

Comment:
Ideally the `Bearer` string should be taken from the capability

Reply:
@magopian can you open a follow-up issue for this please?

Reply:
opened #543